test/plots/areaPlotTests.ts

```diff
@@ -2,162 +2,177 @@
       plot.renderTo(svg);
       assert.strictEqual(yScale.domain()[0], constantY0, "y Scale doesn't pad beyond 0 when used in a Plots.Area");
       svg.remove();
     });
-  });
 
-  describe("AreaPlot", () => {
-    let svg: d3.Selection<void>;
-    let xScale: Plottable.Scales.Linear;
-    let yScale: Plottable.Scales.Linear;
-    let xAccessor: any;
-    let yAccessor: any;
-    let y0Accessor: any;
-    let colorAccessor: any;
-    let fillAccessor: any;
-    let twoPointData = [{foo: 0, bar: 0}, {foo: 1, bar: 1}];
-    let simpleDataset: Plottable.Dataset;
-    let areaPlot: Plottable.Plots.Area<number>;
-    let renderArea: d3.Selection<void>;
-
-    before(() => {
-      xScale = new Plottable.Scales.Linear();
-      xScale.domain([0, 1]);
-      yScale = new Plottable.Scales.Linear();
-      yScale.domain([0, 1]);
-      xAccessor = (d: any) => d.foo;
-      yAccessor = (d: any) => d.bar;
-      y0Accessor = () => 0;
-      colorAccessor = (d: any, i: number, m: any) => d3.rgb(d.foo, d.bar, i).toString();
-      fillAccessor = () => "steelblue";
-    });
+    describe("Rendering", () => {
+      let svg: d3.Selection<void>;
+      let xScale: Plottable.Scales.Linear;
+      let yScale: Plottable.Scales.Linear;
+      let twoPointData = [{x: 0, y: 0}, {x: 1, y: 1}];
+      let simpleDataset: Plottable.Dataset;
+      let areaPlot: Plottable.Plots.Area<number>;
+      let renderArea: d3.Selection<void>;
+
+      beforeEach(() => {
+        xScale = new Plottable.Scales.Linear();
+        yScale = new Plottable.Scales.Linear();
+        svg = TestMethods.generateSVG();
+        simpleDataset = new Plottable.Dataset(twoPointData);
+        areaPlot = new Plottable.Plots.Area<number>();
+        areaPlot.addDataset(simpleDataset);
+        areaPlot.x((d: any) => d.x, xScale)
+                .y((d: any) => d.y, yScale);
+        areaPlot.y0(() => 0)
+                .attr("fill", () => "steelblue")
+                .attr("stroke", (d, i) => d3.rgb(d.x, d.y, i).toString())
+                .renderTo(svg);
+        renderArea = areaPlot.content();
+      });
 
-    beforeEach(() => {
-      svg = TestMethods.generateSVG(500, 500);
-      simpleDataset = new Plottable.Dataset(twoPointData);
-      areaPlot = new Plottable.Plots.Area<number>();
-      areaPlot.addDataset(simpleDataset);
-      areaPlot.x(xAccessor, xScale)
-              .y(yAccessor, yScale);
-      areaPlot.y0(y0Accessor)
-              .attr("fill", fillAccessor)
-              .attr("stroke", colorAccessor)
-              .renderTo(svg);
-      renderArea = (<any> areaPlot)._renderArea;
-    });
+      it("draws area and line correctly", () => {
+        let areaPath = renderArea.select(".area");
+        let pointsInArea = getPointsInArea(areaPlot, twoPointData);
+        TestMethods.assertLinePathEqualToDataPoints(areaPath.attr("d"), pointsInArea, xScale, yScale);
+        assert.strictEqual(areaPath.attr("fill"), "steelblue", "area fill was set correctly");
```

Comment:
Let's factor our the "steelblue" part

Also, why do we want to test the "fill" attribute specifically?